Source/Core/DolphinQt/GameList/GameListModel.cpp

```diff
@@ -185,6 +187,22 @@ QVariant GameListModel::data(const QModelIndex& index, int role) const
     {
       const QString compression = QString::fromStdString(game.GetCompressionMethod());
       return compression.isEmpty() ? tr("No Compression") : compression;
+    }
+    break;
+  case Column::TimePlayed:
+    if (role == Qt::DisplayRole || role == SORT_ROLE)
+    {
+      TimePlayed timer(game.GetGameID());
+
+      int game_milliseconds = timer.GetTimePlayed();
+      // nanoseconds to minutes conversion
+      int game_minutes = game_milliseconds / (60000);
+      int game_hours = game_minutes / 60;
+      int remaining_game_minutes = game_minutes % 60;
+
+      return QString::fromStdString(std::to_string(game_hours) + " Hours, " + std::to_string(remaining_game_minutes) + " Minutes");
```

Comment:
`return QString::fromStdString(fmt::format("{0:H} Hours, {0:M} Minutes", std::chrono::milliseconds{timer_GetTimePlayed()}));`

It will be zero padded but it seems `fmt` may support `-` specifier soon.

Reply:
I tried out this syntax but ended up revising the original display instead since I wanted to follow a similar approach to RPCS3